docs/site/content/docs/tmc-register.md

```diff
@@ -0,0 +1,53 @@
+# Register a Management Cluster in Tanzu Mission Control
+
+You can register a management cluster in VMWare Tanzu Mission Control.
+
+VMware Tanzu Mission Control is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds.
+
+
+1. Log into Tanzu Mission Control, for more information, see [Log In to the Tanzu Mission Control Console](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-855A8998-E19A-46AC-A833-12C347486EF7.html).
+1. In the Administration tab, click REGISTER MANAGEMENT CLUSTERS and select Tanzu Kubernetes Grid. This launches a wizard.
+
+1. **Name and assign:** Enter a name and group for your management cluster and click Next.
+
+1. **Proxy Configuration:** Ensure Set proxy for management cluster is set to No and click Next.
+
+1. **Register:** Copy the registration URL.
+
+1. Create a new file called `tmc-registration.yaml` and paste the contents of the TMC registration url from the previous step into this new file.
+
+    ```sh
+    vim tmc-registration.yaml
+    ```
+
+1. Before you proceed, ensure you context is set to the management cluster. Run:
+
+    ```sh
+    tanzu mc kubeconfig get --admin 
+    ```
+
+    Copy the run the output, for example
+
+    ```sh
+    kubectl config  use-context mgmttest-admin@mgmttest
+    ```
+
+1. Now that you are interacting with the management cluster, apply the `tmc-register.yaml` file:
+
+    ```sh
+    apply -f tmc-register.yaml
+    ```
+
+1. (Optional)  To watch the Tanzu Mission Control pods get created, run:
+
+    ```sh
+    kubectl get po -A
+    ```
+
+    From the output copy the `vmware-system-tmc` namespace and run
+
+    ```sh
+    kubectl -n vmware-system-tmc get po 
+    ```
+
+1. Once all the pods associated with Tanzu Mission Control are running, in the Tanzu Mission Control console, you should be able to click View the Management Cluster, and click Verify Connection.
```

Comment:
At the end here, let's add a "next steps" or something similar that points to the official TMC docs and tells users how to do new things.